### PR TITLE
Easier handling of ExternalEvent by allow `ExternalEvent.event_name`

### PR DESCRIPTION
**Pull request by steve02081504**
same as https://github.com/Taromati2/ghost/commit/0fee9e70b6423e57e40bd89d8098efbb0b6213c5

### CODE REVIEW

File: yaya_base/shiori3.dic [hunk 1]
```diff
@@ -499,6 +475,25 @@ request
 		}
 	}
 
+	if SHIORI3FW.Eventid == 'OnAiTalk' || SHIORI3FW.Eventid == 'OnAITalk' {
+		SHIORI3FW.Eventid = 'OnAITalkNewEvent'
+	}
+
+	if TOLOWER(SHIORI3FW.SecurityLevel) == 'external'{
+		SHIORI3FW.Eventid = "ExternalEvent." + SHIORI3FW.Eventid
```
Comment (nikolat) [inline yaya_base/shiori3.dic:483]: use single quote: 
`SHIORI3FW.Eventid = 'ExternalEvent.' + SHIORI3FW.Eventid`

File: yaya_base/shiori3.dic [hunk 2]
```diff
@@ -499,6 +475,25 @@ request
 		}
 	}
 
+	if SHIORI3FW.Eventid == 'OnAiTalk' || SHIORI3FW.Eventid == 'OnAITalk' {
+		SHIORI3FW.Eventid = 'OnAITalkNewEvent'
+	}
+
+	if TOLOWER(SHIORI3FW.SecurityLevel) == 'external'{
```
Comment (nikolat) [inline yaya_base/shiori3.dic:482]: add space before `{`: 
`if TOLOWER(SHIORI3FW.SecurityLevel) == 'external' {`

**Review by nikolat — COMMENTED** [review]
I think it is a useful improvement idea. 👍

File: yaya_base/shiori3.dic (hunk 1 above)
Comment (steve02081504) [reply yaya_base/shiori3.dic]: ohhhh
thanks for check!
fixed at https://github.com/YAYA-shiori/yaya-dic/commit/910e5b67905afc522a19135985982cf1edc1b763

File: yaya_base/shiori3.dic (hunk 2 above)
Comment (steve02081504) [reply yaya_base/shiori3.dic]: fixed at https://github.com/YAYA-shiori/yaya-dic/commit/910e5b67905afc522a19135985982cf1edc1b763

**Review by nikolat — APPROVED** [review]
👍

**Review by ponapalt — APPROVED** [review]

**Pull request merged**